docs/4.x/tagmanager/embedding.md

```diff
@@ -23,10 +23,12 @@ The code looks as follows:
 ```html
 <!-- Matomo Tag Manager -->
 <script type="text/javascript">
-    window._mtm = window._mtm || [];
-    window._mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+  window._mtm = window._mtm || [];
+  window._mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+  (function() {
     var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
-    g.type='text/javascript'; g.async=true; g.src='https://{$MATOMO_URL}/js/container_{$CONTAINER}.js'; s.parentNode.insertBefore(g,s);
+    g.type='text/javascript'; g.async=true; g.src='https://{$MATOMO_URL}/js/container_{$CONTAINER_ID}.js'; s.parentNode.insertBefore(g,s);
```

Comment:
You need to update the references to `{$CONTAINER}` a few lines below and somewhere near the end of that file or change it back to `{$CONTAINER}` here again. Same applies to 5.x file

Reply:
Thank you @sgiehl . I will make things more consistent.